media-proxy/include/mesh/DEVELOPMENT.md

```diff
@@ -71,6 +71,11 @@ public:
                                   uint32_t& sent) {
         printf("Received %s (%u)\n", (char *)ptr, sz);
     }
+
```

Comment:
Don't add a new method for that. Set the `configured` state in the emulated tx/rx constructor instead.